src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,9 @@
+@import "../utils/variables";
+
+.product-catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 1000px;
+  padding: 50px 0;
+}
```

Comment:
Remember about styles properties order (number 12 in checklist)

Reply:
Fix this!